elements/pfe-sass/extends/_extends.scss

```diff
@@ -47,3 +47,187 @@
     color: pfe-fetch(text) !important;
   }
 }
+
+// NOTES: 
+// - The interpoliation is for the benfefit of older systems
+// - These do not make use of the pfe-typography mixin because it 
+//   does not include the --pf-c-title--m CSS variables in the stack
+
+// TITLES
+%pfe-title--6xl {
+    font-family: #{pfe-var(font-family--heading)};
+    font-size:   #{pfe-var(title--m-6xl--FontSize)};
+    line-height: #{pfe-var(line-height--heading)};
+    font-weight: #{pfe-var(font-weight--light)};
+    &:not(:last-child) {
+        margin-bottom: pfe-var(content-spacer--heading--md);
+    }
+}
+//%pfe-headline-xl  
+%pfe-title--5xl {
+    font-family: #{pfe-var(font-family--heading)};
+    font-size:   #{pfe-var(title--m-5xl--FontSize)};
+    line-height: #{pfe-var(line-height--heading)};
+    font-weight: #{pfe-var(font-weight--light)};
+    &:not(:last-child) {
+        margin-bottom: pfe-var(content-spacer--heading--md);
+    }
+}
+//%pfe-headline-alpha  
+%pfe-title--4xl {
+    font-family: #{pfe-var(font-family--heading)};
+    font-size:   #{pfe-var(title--m-4xl--FontSize)};
+    line-height: #{pfe-var(line-height--heading)};
```

Comment:
@starryeyez024 I noticed when writing test cases that the deprecated line height is 1.3 but we're now using 1.2. Is this intentional?

Reply:
good point, yeah we have these vars in the maps typography file:
```
  line-height--heading:      1.2,
  line-height:               1.3,
```
its just that only the top two largest headlines should use the 1.2. Will fix!